Dockerfile

```diff
@@ -1,8 +1,13 @@
-FROM yunspace/alpine-golang
+FROM golang:alpine
 
-ADD build/codeclimate-gofmt /usr/src/app/
+MAINTAINER Code Climate <hello@codeclimate.com>
 
 RUN adduser -u 9000 -D app
-USER app
+
+WORKDIR /code
+
+COPY build /usr/src/app
```

Comment:
Comment chain got lost in a diff again.

Do we actually need anything in `build` other than the `codeclimate-gofmt` bin? If not, can we just copy that bin and not whatever other junk also ends up in `build`?